third_party/proton/test/test_profile.py

```diff
@@ -97,8 +94,10 @@ def fn():
         torch.cuda.synchronize()
         proton.exit_scope()
         proton.finalize()
+
+
         data = json.load(f)
-        # CUDA graph may also invoke additional kernels to reset outputs
+        # CUDA/HIP graph may also invoke additional kernels to reset outputs
```

Comment:
Based on your test case, HIP graph doesn't add additional kernels?

Reply:
That's a good point. I originally thought this was an 0 vs 1 indexing difference but looking at it again want to dig a bit deeper.